Code/Engine/GameEngine/Physics/Implementation/CollisionFilter.cpp

```diff
@@ -23,7 +23,7 @@ void ezCollisionFilterConfig::SetGroupName(ezUInt32 uiGroup, ezStringView sName)
 
 ezStringView ezCollisionFilterConfig::GetGroupName(ezUInt32 uiGroup) const
 {
-  return m_GroupNames[uiGroup];
+  return ezStringView((const char*)m_GroupNames[uiGroup]);
```

Comment:
fix for a nasty issue: the whole 32 byte array was given to the ezStringView, so it included \0 etc

Reply:
I would prefer ezString m_GroupNames[32], don't quite understand why this has to be a raw char buffer.

Reply:
fixed